codegen/types/test/test_method.py

```diff
@@ -138,3 +142,43 @@ def test_from_dict(self) -> None:
             # Act and assert
             with self.assertRaises(InvalidMethodSpecError):
                 Method.from_dict(this_data)
+
+    def test_get_docs_if_available(self) -> None:
+        # Arrange
+        test_method = self.methods["get"]
+        test_docs_dir = pathlib.Path("docs")
+        expected_api_path = test_docs_dir / "get.md"
+        expected_response_path = test_docs_dir / "get.response.md"
```

Comment:
[dust] I usually find it easier to find things when I don't have to deal with both qualified and unqualified names. I could see either of these filename groups working for me:
```
get.summary.md  # docs site link and other notes
get.request.md  # field types and descriptions
get.response.md
```
```
get.request.md # docs site link + notes + types
get.response.md
```

But I could also see that `get.md` _is_ `get.summary.md` and/or `get.request.md`.

Reply:
That's a good point. I renamed `get.md` to `get.request.md`